README.md

```diff
@@ -13,11 +13,27 @@ The GenAIInfra repository is organized under four main directories, which includ
 | `manifests`         | Manifests for deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes and on Docker Compose. |
 | `scripts`           | Scripts for testing, tools for facilitate OPEA deployment, and etc.                                                         |
 
-## Usages
+## Prerequisite
+
+GenAIInfra uses Kubernetes as the cloud native infrastructure. Please follow the following steps to prepare the Kubernetes environment.
+
+### Setup Kubernetes cluster
+
+Please follow [Kubernetes official setup guide](https://kubernetes.io/docs/setup/) to setup the Kubernetes itself. We recommend to use at least Kubernetes v1.27.
+
+### (Optional) To run GenAIInfra on [Intel Gaudi](https://habana.ai/products/) product:
 
-### Prerequisites
+The following steps are optional. They're only required if you want to run the workloads on Intel Gaudi product.
 
-GenAIInfra uses [Kubernetes](https://kubernetes.io/) as the cloud native infrastructure. You will need the access to either managed kubernetes services (e.g., EKS,ASK,GKE and etc) or self managed kubernetes.
+1. Please check the [support matrix](https://docs.habana.ai/en/latest/Support_Matrix/Support_Matrix.html) to make sure that environment meets the requirements.
+
+2. [Install Intel Gaudi software stack](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#driver-fw-install-bare).
+
+3. [Install and setup container runtime](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#set-up-container-usage), based on the container runtime used by Kubernetes.
```

Comment:
Suggest to copy the instructions of configuring containerd. The original link default is docker setup, someone may have issues to find the correct place.
sudo tee /etc/containerd/config.toml <<EOF
disabled_plugins = []
version = 2

[plugins]
  [plugins."io.containerd.grpc.v1.cri"]
    [plugins."io.containerd.grpc.v1.cri".containerd]
      default_runtime_name = "habana"
      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.habana]
          runtime_type = "io.containerd.runc.v2"
          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.habana.options]
            BinaryName = "/usr/bin/habana-container-runtime"
  [plugins."io.containerd.runtime.v1.linux"]
    runtime = "habana-container-runtime"
EOF

Reply:
K8S now supports both containerd and docker-ce container runtime. I've verified on ubuntu 22.04 for both of them.

I've added the NOTE paragraph for users to configure the right container runtime down